tests/testHSCColorterms.py

```diff
@@ -0,0 +1,97 @@
+import os
+import numbers
+import unittest
+
+import lsst.utils.tests as utilsTests
```

Comment:
I'm embarrassed to admit that I thought I had but then it turns out I was waiting for this week's DM highlights (the changes were approved last Thursday).

As an aside there were no previous instructions for testing of Python code in the developer guide. These things sort of evolved.